server/postgresql/seed/dishes_seed.js

```diff
@@ -0,0 +1,27 @@
+const faker = require('faker');
+const path = require('path');
+const fs = require('fs');
+
+
+    let titles = "id,dish_name,dish_price,summary,photo_count,review_count,restaurant_id \n";
+    let data = "";
+    for (let i = 0; i < 100000; i++) {
+        let id = i + ',';
```

Comment:
if you're using serial primary key you don't need to generate the id, it will automatically be added